pennylane/gradients/pulse_gradient.py

```diff
@@ -412,14 +412,14 @@ def ansatz(params):
 
     >>> params = [jax.numpy.array(0.4), jax.numpy.array(1.3)]
     >>> qnode(params)
-    Array(-0.0905377, dtype=float32)
+    Array(-0.0905377, dtype=float64)
 
     And as we registered the differentiation method :func:`~.stoch_pulse_grad`,
     we can compute its gradient in a hardware compatible manner:
 
     >>> jax.grad(qnode)(params)
-    [Array(0.00054451, dtype=float32, weak_type=True),
-     Array(-0.05673098, dtype=float32, weak_type=True)] # results may differ
```

Comment:
Did you consciously remove the comment? :)

Reply:
Oh thanks, no! Missed that detail!